From e4c2ac72e20043626bf6e5308e92dfab5c9d4e26 Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@nxp.com>
Date: Fri, 4 Aug 2017 15:05:16 +0800
Subject: [PATCH 2331/5242] MLK-16136-10 ARM64: dts: imx8mq-evk: enable HDMI
 display by default

commit  3e5ff330bed4cb9c775d3d4d48a0d6f2a5d426ed from
https://source.codeaurora.org/external/imx/linux-imx.git

The default display path on imx8mq-evk board is
'DCSS --> HDMI'. So enable the HDMI controller
by default.

Signed-off-by: Fancy Fang <chen.fang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
index 030312c..252f10b 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
@@ -444,7 +444,7 @@
 
 &hdmi {
 	disp-dev = "hdmi_disp";
-	status = "disabled";
+	status = "okay";
 };
 
 &i2c2 {
-- 
1.7.9.5

